app/src/main/assets/debug/DebugActivity.java

```diff
@@ -1,76 +1,63 @@
 package <?package_name?>;
 
+import android.annotation.SuppressLint;
 import android.app.Activity;
 import android.app.AlertDialog;
+import android.content.Context;
 import android.content.DialogInterface;
-import android.content.Intent;
 import android.os.Bundle;
-import android.util.Log;
 import android.widget.TextView;
 
 public class DebugActivity extends Activity {
-
-    private String[] exceptionTypes = {
-            "StringIndexOutOfBoundsException",
-            "IndexOutOfBoundsException",
-            "ArithmeticException",
-            "NumberFormatException",
-            "ActivityNotFoundException"
-    };
-
-    private String[] exceptionMessages = {
-            "Invalid string operation\n",
-            "Invalid list operation\n",
-            "Invalid arithmetical operation\n",
-            "Invalid toNumber block operation\n",
-            "Invalid intent operation"
-    };
```

Comment:
I really don't think these simplified error messages should be removed.

Reply:
It doesn't help anymore because of the bizzare custom blocks people uses these days.
They were helpful in vanila sw sure, but I don't think the extra stuffs are needed anymore.
But you can add them back if you think those are necessary.
Imo they're useless.

Reply:
@khaled-0 what would you say now?

Reply:
you're intentionally crashing the app 
it's causing to crash on every startup.

I dont think this behaivour can be avoided with what this pr does for now.
unless there's a way to not continue the app if there's a crash